services/gundeck/src/Gundeck/Push/Data.hs

```diff
@@ -47,6 +48,12 @@ insert u t a p e o c = retry x5 $ write q (params LocalQuorum (u, t, a, p, e, o,
     q :: PrepQuery W (UserId, Transport, AppName, Token, EndpointArn, ConnId, ClientId) ()
     q = "insert into user_push (usr, transport, app, ptoken, arn, connection, client) values (?, ?, ?, ?, ?, ?, ?)"
 
+updateArn :: MonadClient m => UserId -> Transport -> AppName -> Token -> EndpointArn -> m ()
+updateArn u t a p arn = retry x5 $ write q (params LocalQuorum (arn, u, t, a, p))
```

Comment:
I guess it's from _p_ush token. I'm totally indifferent to variable names, unfortunately, so I can change it and trigger all people that like consistent variable names for the same things within one module by changing the variable names for this one function. In a way, I have already without thinking, since I called my variable `arn` instead of `e` for endpoint as in the function before. 😞

Reply:
https://github.com/wireapp/wire-server/pull/3162/commits/6c0dbcd31f211ab3ebc3d7224ed8afe277f99d7a